backend/tests/userTests.test.js

```diff
@@ -0,0 +1,89 @@
+// M6 Q3
+//import {jest} from '@jest/globals';
+//import axios from 'axios';
+var axios = require('axios');
+const { ObjectID } = require('bson');
+
+const UserStoreURL = 'http://20.53.224.7:8082';
+const UserDBURL = 'http://20.53.224.7:8081';
+
+jest.mock('axios');
```

Comment:
Codacy has a fix for the issue: [Replace `'axios'` with `"axios"`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
jest.mock("axios");
```